src/main/kotlin/org/amshove/kluent/Collections.kt

```diff
@@ -1,26 +1,26 @@
 package org.amshove.kluent
 
-import org.junit.Assert.*
+import java.util.*
 
 infix fun <T> Array<T>.shouldContain(theThing: T) = if (this.contains(theThing)) Unit else fail("$this should contain $theThing", "$theThing", join(this))
 
-infix fun <T> Array<T>.shouldContainSome(things: Array<T>) = assertTrue("Expected $this to contain at least one of $things", this.any { things.contains(it) })
+infix fun <T> Array<T>.shouldContainSome(things: Array<T>) = assert(this.any { things.contains(it) }, {"Expected $this to contain at least one of $things"})
 
-infix fun <T> Array<T>.shouldContainNone(things: Array<T>) = assertTrue("Expected $this to contain none of $things", this.none { things.contains(it) })
+infix fun <T> Array<T>.shouldContainNone(things: Array<T>) = assert(this.none { things.contains(it) }, {"Expected $this to contain none of $things"})
 
 infix fun <T> Array<T>.shouldContainAll(things: Array<T>) = things.forEach { shouldContain(it) }
 
 infix fun <T> Array<T>.shouldNotContain(theThing: T) = if (!this.contains(theThing)) Unit else fail("$this should not contain $theThing", "the Array to not contain $theThing", join(this))
 
 infix fun <T> Array<T>.shouldNotContainAny(things: Array<T>) = things.forEach { shouldNotContain(it) }
 
-infix fun <T> Array<T>?.shouldEqual(theOther: Array<T>?) = assertArrayEquals(theOther, this)
+infix fun <T> Array<T>?.shouldEqual(theOther: Array<T>?) = assert(Arrays.equals(this, theOther))
```

Comment:
I think there should be a way to compare arrays in plain Kotlin, without the `java.util.Arrays` import

Reply:
`arr1 contentEquals arr2` does exist but it doesn't work since the arrays are optional.

Reply:
You can go with a helper function:

```kotlin
fun arraysEqual(a1: Array<Int>?, a2: Array<Int>?): Boolean {
  if (a1 == null || a2 == null) return false
  return a1 contentEquals a2
}
```

The behavior is consistent with `Arrays.equals()`